build.bat

```diff
@@ -19,7 +19,7 @@ IF NOT "%APPVEYOR_BUILD_VERSION%"=="" (
 )
 
 IF "%PACKAGE_VERSION%"=="" (
-	SET PACKAGE_VERSION=0.0.1
+	SET PACKAGE_VERSION=1.0.5
```

Comment:
This version only affects local debugging versions since the "official" version is controlled and published by the CI/CD build (AppVeyor).